### PR TITLE
Add copy appointments details to clipboard button

### PR DESCRIPTION
**Pull request by yedidya03**
- adding copy appointment details button
- fixing alignment of buttons to be horizontal

### CODE REVIEW

File: packages/coordinator/src/screens/manageAppointmentsScreen/ManageAppointmentsTableConfig.tsx [hunk 1]
```diff
@@ -84,6 +85,21 @@ const GetExpandedColumns = (
...
+          iconUrl: copyIcon,
+          onClick: () => {
+            let bookingDate = appointment.appointmentTimeMilis
+              ? DateUtils.ToDateString(appointment.appointmentTimeMilis)
+              : "missing-date";
+            let copyString = `${appointment.donorName}, ${String(
+              appointment.donorPhoneNumber
+            )}, ${bookingDate}`;
+            navigator.clipboard.writeText(copyString);
+            console.log(copyString);
```
Comment (yaron1m) [inline packages/coordinator/src/screens/manageAppointmentsScreen/ManageAppointmentsTableConfig.tsx:99]: Remove the log print

File: packages/coordinator/src/screens/manageAppointmentsScreen/ManageAppointmentsTableConfig.tsx [hunk 2]
```diff
@@ -84,6 +85,21 @@ const GetExpandedColumns = (
       }[] = [];
 
       if (appointment.booked) {
+        buttons.push({
+          tooltip: "העתק פרטים",
+          iconUrl: copyIcon,
+          onClick: () => {
+            let bookingDate = appointment.appointmentTimeMilis
+              ? DateUtils.ToDateString(appointment.appointmentTimeMilis)
+              : "missing-date";
```
Comment (yaron1m) [inline packages/coordinator/src/screens/manageAppointmentsScreen/ManageAppointmentsTableConfig.tsx:94]: I think we should have an empty string, and not `missing-date`

File: packages/coordinator/src/screens/manageAppointmentsScreen/ManageAppointmentsTableConfig.tsx [hunk 3]
```diff
@@ -84,6 +85,21 @@ const GetExpandedColumns = (
       }[] = [];
 
       if (appointment.booked) {
+        buttons.push({
+          tooltip: "העתק פרטים",
```
Comment (yaron1m) [inline packages/coordinator/src/screens/manageAppointmentsScreen/ManageAppointmentsTableConfig.tsx:89]: העתקת
To make this gender neutral

File: packages/coordinator/src/screens/manageAppointmentsScreen/CoordinatorAppointmentsGrouper.ts [hunk 4]
```diff
@@ -21,6 +21,7 @@ export type ManagedAppointment = {
   appointmentId: string;
   booked: boolean;
   donorName?: string;
+  appointmentTimeMilis?: number;
```
Comment (yaron1m) [inline packages/coordinator/src/screens/manageAppointmentsScreen/CoordinatorAppointmentsGrouper.ts:24]: Please stay consistent with the naming - call this `donationStartTimeMillis`

**Review by yaron1m — APPROVED** [review]
Please add a screenshot of the changed parts

Great job!

**Comment by yedidya03** [comment]
image

**Comment by yaron1m** [comment]
Looks like the columns don't look well after the horizontal change.

I.e.the blood type is not centered, and the booking date is not aligned with the column header.

Either fix this or remove the horizontal change from this PR

**Pull request merged**